src/schema/saleable.ts

```diff
@@ -0,0 +1,29 @@
+import Dimensions from "./dimensions"
+import { GraphQLInterfaceType, GraphQLBoolean, GraphQLString } from "graphql"
+
+export const Saleable = new GraphQLInterfaceType({
```

Comment:
`Sellable` might be a better name than `Saleable`?

Reply:
I went down a little 🐰 🕳 for this one. E.g. from https://www.englishforums.com/English/SellableVersusSaleableVesus-Salable/nrchc/post.htm:

> I disagree. I have two business degrees, minored in linguistics, and have been working in the retail operations and marketing field all my adult life. Salable, saleable, and sellable are often pronounced the same and really all mean the same thing-- able to be sold. There is no connotation or implication of profit margin. With that being said, however, in the US "saleable" is BY FAR the most commonly used in writing. When I see that someone has written "salable," I pause to wonder where they might be from. When I see that someone has written "sellable," I know they likely aren't a business pro.

There’s more on e.g. https://forum.wordreference.com/threads/saleable-vs-sellable.523745/